### DISCUSSION TITLE
我的bot无法使用

### DISCUSSION THREAD

**Discussion by zhanming-Chinese** [nonebot/discussions#345] [问答]
我已经安装好了nonebot与go-cq并且成功建立链接，两边都能显示收到qq的信息，但是我的bot貌似无法发送信息，例如：我说/echo 你好，世界，它无法输出；我安装了一个可以说'6'的插件，它也无法发送。请问我该怎么解决

**Comment by yanyongyu** [comment]
日志截图

**Reply by zhanming-Chinese** [reply]
使用nb run启动的go-cq和nonebot日志
![image](https://user-images.githubusercontent.com/62282155/226165499-fbeffcf2-6d29-48b5-b71f-c5465c7b30fd.png)
![image](https://user-images.githubusercontent.com/62282155/226165516-6f5da35e-45eb-488f-be75-b5ba488ccff2.png)

使用python启动的go-cq和nonebot日志以及python代码
![image](https://user-images.githubusercontent.com/62282155/226165641-bd5eb73e-fad9-478a-b539-6a5d78387349.png)
![image](https://user-images.githubusercontent.com/62282155/226165651-dfa93208-ea9a-4d33-9256-2e2c3d439a80.png)
```
import nonebot

if __name__ == '__main__':
    nonebot.init()
    nonebot.load_builtin_plugins()
    nonebot.run(host='127.0.0.1', port=56345)
```

**Reply by zhanming-Chinese** [reply]
> 日志截图

请问我还需要提供更多信息吗

**Reply by yanyongyu** [reply]
你的发送echo消息的日志

**Comment by zhanming-Chinese** [comment]
nb run方式启动：
![image](https://user-images.githubusercontent.com/62282155/226169667-76c179ce-023c-4e45-a055-ef6a1c79e5bc.png)
![image](https://user-images.githubusercontent.com/62282155/226169678-d74861b1-5d34-4429-8d70-53f608e2ec82.png)
python启动：
![image](https://user-images.githubusercontent.com/62282155/226169740-f889ed09-d330-4555-b395-ecf0704fd80c.png)
![image](https://user-images.githubusercontent.com/62282155/226169734-8ed79021-4aa0-42dc-9f6a-2596693a562f.png)

**Reply by yanyongyu** [reply]
你的command_start配置为空

**Reply by maoxig** [reply]
哥，端口写错了没对上，仔细看看俩个数完全不同啊喂

**Reply by maoxig** [reply]
一个是56435一个是56345

**Reply by zhanming-Chinese** [reply]
> 你的command_start配置为空

我把command_start在.env内设置为了"\"，把command_ser设置为了” “，并且把https://docs.nonebot.dev/guide/command.html#%E7%BC%96%E5%86%99%E7%9C%9F%E6%AD%A3%E7%9A%84%E5%86%85%E5%AE%B9中天气预报的代码复制到.py装进了src/plugins用以测试，再次运行时仍然无法收到bot发送的内容
![image](https://user-images.githubusercontent.com/62282155/230069664-bda1a48c-3c67-4804-a587-aa34919260f7.png)
其中，python还有如下报错
![image](https://user-images.githubusercontent.com/62282155/230069762-8e504b72-14ce-4f2c-a3c0-189d22e8e945.png)

**Reply by yanyongyu** [reply]
`CommandSession`是nb1的东西
